v1/linkml-schemas/name.yaml

```diff
@@ -17,34 +17,37 @@ default_range: string
 
 classes:
   NameClass:
+    description: Name object that represnts a user's claimed identity.  This will have a list of name parts.  Usually this will contain one or more [GivenNameClass](../GivenNameClass) name parts and one [FamilyNameClass](../FamilyNameClass) name part for a specific period of validity.
     mixins:
       - ValidityClass
-    attributes:
-      nameParts:
-        range: NamePartClass
-        multivalued: true
-        inlined_as_list: true
+    slots:
+      - nameParts
+      
   NamePartClass:
+    description: Abstract class to define a name part value and name part type for an optional validity period.
     abstract: false  # has to be false otherwise JSON-Schema doesn't include it, and validation fails :(
     mixins:
       - ValidityClass
     slots:
       - value
-    attributes:
-      type:
-        range: NamePartType
-        multivalued: false
+      - type
+
   GivenNameClass:
+    description: Name part object that represents a Given Name.  There may be multiple [NamePart](../NamePartClass) objects with a [NamePartType](../NamePartType) of `GivenName` within the representation of a [Name](../NameClass) for a specific period of validity.  These will appear in the order in which the user specified or the order in which they were read from official documentation.
     is_a: NamePartClass
-    attributes:
-      type:
-        equals_string: GivenName
 
-  FamilyNameClass:
+  FamilyNameClass: 
+    description: Name part object that represents a Family Name.  There will only ever be a single [NamePart](../NamePartClass) object with a [NamePartType](../NamePartType) of `FamilyName` within the representation of a [Name](../NameClass) for a specific period of validity.  This property will contain the user specified family name or the family name that was read from official documentation.
```

Comment:
This isn't what we're trying to say, I think, let's chat

Reply:
I agree that this is not in line with RFC11, but this is the way the current implementation is.  There is no code in any of the CRIs to do anything other than return one name part as a family name.  The places where names are currently being given are DCMAW (Passport and Driving License) we know they don't have code to split Family Name Parts.  The other place is in the web journey through the passport CRI and when running this through staging with a space in the family name field we end up with:

```
{
  "sub": "urn:uuid:fb1698db-5675-4eee-a737-c67298b52b93",
  "aud": "https://identity.staging.account.gov.uk",
  "nbf": 1675114802,
  "iss": "https://review-p.staging.account.gov.uk",
  "exp": 1690753202,
  "vc": {
    "evidence": [
      {
        "validityScore": 0,
        "strengthScore": 4,
        "ci": [
          "D02"
        ],
        "txn": "66368711-f788-442a-b637-151c65847447",
        "type": "IdentityCheck"
      }
    ],
    "credentialSubject": {
      "passport": [
        {
          "expiryDate": "2024-09-09",
          "icaoIssuerCode": "GBR",
          "documentNumber": "123456789"
        }
      ],
      "name": [
        {
          "nameParts": [
            {
              "type": "GivenName",
              "value": "JAMES"
            },
            {
              "type": "GivenName",
              "value": "WILLIAM"
            },
            {
              "type": "FamilyName",
              "value": "SMITH WOODS"
            }
          ]
        }
      ],
      "birthDate": [
        {
          "value": "1980-10-01"
        }
      ]
    },
    "type": [
      "VerifiableCredential",
      "IdentityCheckCredential"
    ]
  }
}
```

We need to decide if we think this is tech debt and we should be implementing Family Name splitting to align with RFC11 or we need to change the main wording on RFC11 to bring it in line with the current implementation.  We sort of decided yesterday not to implement Family Name splitting in the DCMAW passport journey as we know the MRZ read can give us problems and it may help us spot them, but it seems like currently there is never going to be any family name splitting anywhere regardless of an MRZ read document or not as it has never been implemented.

Reply:
Hopefully latest check-in now aligns this to the wording updates on RFC11 and we have reserved the right to use multiple FamilyName name parts in future.